way_to_home/custom_user/views.py

```diff
@@ -101,17 +115,17 @@ def signin_google(request):
                                    state=STATE, scope=SCOPE)
     authorization_code = request.GET.get("code")
     if not authorization_code:
-        return HttpResponse("Code doesn't exist", status=400)
+        return HttpResponse("Code not received", status=400)
```

Comment:
```suggestion
        return HttpResponse("Code not received", status=400)
```
I think you can use response constant `RESPONSE_400_INVALID_DATA`

Reply:
Agree, done